<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-3BDBSD1" failures="0" tests="3" name="com.cdn.testcases.AddCustomersTest" time="3.131" errors="0" timestamp="2025-02-11T22:13:44 IST" skipped="1">
  <testcase classname="com.cdn.testcases.AddCustomersTest" name="addCustomerTest" time="0.014">
    <skipped/>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="com.cdn.testcases.AddCustomersTest" name="addCustomerTest" time="1.285"/>
  <testcase classname="com.cdn.testcases.AddCustomersTest" name="addCustomerTest" time="1.832"/>
  <system-out>
    <![CDATA[Verify failure: expected [xyz] but found [abc]
</br>
<a target="_blank" href=Tue_Feb_11_22_13_35_IST_2025.jpg><img height=200 width=200 src=Tue_Feb_11_22_13_35_IST_2025.jpg></img></a>
</br>]]>
  </system-out>
</testsuite> <!-- com.cdn.testcases.AddCustomersTest -->
